tests/input_files/beam_ifpack.xml

```diff
@@ -0,0 +1,7 @@
+<ParameterList name="Ifpack">
+  <!-- General settings for Ifpack preconditioners -->
+  <Parameter name="Preconditioner type" type="string" value="ILU"/>
+  <Parameter name="Overlap" type="int" value="1"/>
+  <!-- Level-of-fill setting for Ifpack ILU preconditioner -->
```

Comment:
Such lines could be removed. 
```suggestion
```
I don't have a strong opinion on that, but they seem not to provide additional value.